python/ray/exceptions.py

```diff
@@ -267,6 +267,16 @@ class AsyncioActorExit(RayError):
     pass
 
 
+class RuntimeEnvSetupError(RayError):
+    """Raised when a runtime environment fails to be set up."""
+
+    def __str__(self):
+        return (
+            "The runtime environment for this task or actor failed to be "
+            "installed. Corresponding error logs should have been streamed "
+            "to the driver's STDOUT.")
```

Comment:
where does this happen?

Reply:
They get streamed through the raylet (I'm not actually sure how this works)

Reply:
This existed before my change

Reply:
nvm it's in agent_manager.cc